content/open-firmware-systems.md

```diff
@@ -42,6 +42,7 @@ The source code for our Open EC can be found [here](https://github.com/system76/
 | Adder WS (addw3 and newer) | Yes | Yes |
 | Bonobo WS (bonw13 and older) | No | No |
 | Bonobo WS (bonw14 and newer) | Yes | Yes |
+| Bonobo WS (bonw15-b) | No | No |
```

Comment:
I think it would be better to break this down.
- bonw14 (yes,yes)
- bonw15 (yes,yes)
- bonw15-b (no,no)